abletonosc/clip.py

```diff
@@ -106,9 +108,53 @@ def clip_callback(params: Tuple[Any]) -> Tuple:
             self.osc_server.add_handler("/live/clip/set/%s" % prop,
                                         create_clip_callback(self._set_property, prop))
 
+        def clip_get_notes_helper(clip, params: Tuple[Any] = ()):
+            if clip is None:
+                self.logger.info("Trying to get notes from an empty clip")
+                return ()
+            # Define default values
+            estimated_min_from_time = -16000
```

Comment:
`default_min_from_time` would be a better variable name here

Reply:
@ideoforms updated